programs/cleos/main.cpp

```diff
@@ -1248,20 +1248,30 @@ struct buyram_subcommand {
    string from_str;
    string receiver_str;
    string amount;
+   bool kbytes = false;
 
    buyram_subcommand(CLI::App* actionRoot) {
       auto buyram = actionRoot->add_subcommand("buyram", localized("Buy RAM"));
       buyram->add_option("payer", from_str, localized("The account paying for RAM"))->required();
       buyram->add_option("receiver", receiver_str, localized("The account receiving bought RAM"))->required();
-      buyram->add_option("tokens", amount, localized("The amount of EOS to pay for RAM"))->required();
+      buyram->add_option("tokens", amount, localized("The amount of EOS to pay for RAM, or number of kbytes of RAM if --kbytes is set"))->required();
```

Comment:
Should change "tokens" to something more generic, since it is either tokens to pay or kbytes of RAM to buy.  Maybe use "amount".